checklists/lz_checklist.en.json

```diff
@@ -1434,6 +1434,182 @@
       "guid": "ae514b93-3d45-485e-8112-9bd7ba012f7b",
       "severity": "Medium",
       "link": "https://docs.microsoft.com/security/benchmark/azure/security-control-incident-response"
+    },
+    {
+      "category": "Network Topology and Connectivity",
+      "subcategory": "Front Door",
+      "text": "Avoid combining Azure Traffic Manager and Azure Front Door.",
+      "guid": "062d5839-4d36-402f-bfa4-02811eb936e9",
+      "severity": "High",
+      "link": "https://learn.microsoft.com/azure/frontdoor/best-practices#avoid-combining-traffic-manager-and-front-door"

```

Comment:
only issue I see is that the "link" elements have been to MS Docs reference for all the other items on the checklist.  For Learn modules, we have defined links to them under a "training" element in the JSON objects.

Reply:
@sdolgin Microsoft Docs has just been rebranded to Microsoft Learn. (I know – confusing.) So all docs.microsoft.com URLs are now learn.microsoft.com.

Reply:
thanks for the explanation!